src/imeta.cpp

```diff
@@ -2277,27 +2276,6 @@ usage( const char *subOpt ) {
                 "Add an AVU to a dataobj (-d), collection(-C), resource(-R), ",
                 "or user(-u)",
                 "Example: add -d file1 distance 12 miles",
-                " ",
-                "Admins can also use the command 'adda' (add as admin) to add metadata",
-                "to any collection or dataobj; syntax is the same as 'add'.  Admins are",
-                "also allowed to add user and resource metadata.",
-                ""
-            };
-            for ( i = 0;; i++ ) {
-                if ( strlen( msgs[i] ) == 0 ) {
-                    return 0;
-                }
-                printf( "%s\n", msgs[i] );
-            }
-        }
-        if ( strcmp( subOpt, "adda" ) == 0 ) {
-            char *msgs[] = {
-                " adda -d|C|R|u Name AttName AttValue [AttUnits]  (Add as administrator)",
-                "                                     (same as 'add' but bypasses ACLs)",
-                " ",
-                "Administrators (rodsadmin users) may use this command to add AVUs",
-                "to any dataobj, collection, resource, or user.  The syntax is the same",
-                "as 'imeta add'.",
                 ""
```

Comment:
Since I'm deprecating something as well, this has been on the mind, and I want to make sure I get it right... :)

Are we supposed to remove it from the help text, or leave it in the documentation and leave a message about its being deprecated? It seems we have gone back and forth on this over the years (see `irm -U` for example of not being removed from the help text).

Reply:
good question.

we haven't established a rule around this just yet. this is probably the right time to do so.

for example, below are the potential rules for deprecating features of an icommand (based on this PR):
- remove references to deprecated options from help text
- server should add deprecation message to rError object
- server should log deprecation message

Reply:
i like these three rules.

the twist is on the first one... we left `irm -U` documented/marked as deprecated since there was no other good place to put that notice to the user.   in this case, `adda` seemed out of place to begin with?  and it was documented in the running text above the subcommands, as an option.

happy to have it be more uniform, but also okay with it being a per-item decision about how to inform the human.

Reply:
i've captured the deprecation rule here: irods/irods_docs#124